samples/nrf9160/location/prj.conf

```diff
@@ -16,9 +16,13 @@ CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_HEAP_MEM_POOL_SIZE=8192
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
 
-# Location sample
+# Location library
 CONFIG_LOCATION=y
-#CONFIG_LOCATION_SAMPLE_GNSS_ANTENNA_EXTERNAL=y
+
+# GNSS related modem configuration
+CONFIG_MODEM_GNSS_CFG=y
+# Enable to use an external GNSS antenna
+#CONFIG_MODEM_GNSS_CFG_ANTENNA_EXTERNAL=y
```

Comment:
It would be better to remove the commented line in `prj.conf` and add a couple of lines in the sample documentation on how to use a different antenna.

Reply:
Ok, I've removed the commented lines from configuration files and added the relevant option in the sample documentation instead.